### ISSUE TITLE
The set cost function is ignored! LCB acquisition does not make sense with cost

### ISSUE THREAD

**Issue by timbryant** [SheffieldML/GPyOpt#235]
Hello,
Thanks for your amazing works.When I use the GPyOpt, I found a question,which makes me confused. I defined a func with a parameter r. When I run the BayesianOptimization(), the results shows the func seem not working,and showed :"The set cost function is ignored! LCB acquisition does not make sense with cost".
In the func I defined, the value the func returned is a numpy,which is map(mean average precision),and the parameter didn't influence the map value,but it indeed influence the number which calculate the map, so I wander whether the parameter in the func must influence the returned results or not?
I'm looking forward to your reply!
Thank you very very much!!!!

**Comment by apaleyes** [comment]
Cheers!

Kind of hard to follow the question, to be honest. Can you show us the code to reproduce the issue? Specifically the function you defined and how you are using it
